crates/matrix-sdk/src/account.rs

```diff
@@ -388,10 +391,12 @@ impl Account {
         &self,
         id_server: Option<&str>,
         auth_data: Option<AuthData>,
+        erase_data: bool,
     ) -> Result<deactivate::v3::Response> {
         let request = assign!(deactivate::v3::Request::new(), {
             id_server: id_server.map(ToOwned::to_owned),
             auth: auth_data,
+            erase: erase_data,
```

Comment:
I'm surprised there is no test for this method? Can you check please?

Reply:
There's isn't one, no, but it's all just a network request.

Reply:
added tests in https://github.com/matrix-org/matrix-rust-sdk/pull/4002/commits/e414149d452b9496dec2a539365c723986b39cdc